src/commands/money.js

```diff
@@ -33,9 +35,11 @@ export default class Money {
 
     async ready() {
         try {
-            await this.fetchRates()
+            await this.fetchRates(      )
         } catch (err) {
-            Rin.log.error(err.message || JSON.stringify(err))
+            Rin.log.error(err.message || JSON.stringify(err
+            
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp49z0c0qn/src/commands/money.js
+++ b/tmp/tmp49z0c0qn/src/commands/money.js
@@ -38,7 +38,7 @@
             await this.fetchRates(      )
         } catch (err) {
             Rin.log.error(err.message || JSON.stringify(err
-            
+
             ))
         }
     }
```